components/ActionSheet/examples/ActionSheetSwiftExample.swift

```diff
@@ -14,9 +14,10 @@
 
 import UIKit
 import MaterialComponentsAlpha.MaterialActionSheet
-import MaterialComponentsAlpha.MaterialActionSheet_ActionSheetThemer
+import MaterialComponentsAlpha.MaterialActionSheetThemer
```

Comment:
This line suggests that we are currently transforming some import statements incorrectly